src/Compilers/CSharp/Portable/Symbols/Synthesized/SynthesizedParameterSymbol.cs

```diff
@@ -227,35 +230,39 @@ public override ImmutableArray<CustomModifier> RefCustomModifiers
 
         private sealed class SynthesizedComplexParameterSymbol : SynthesizedParameterSymbolBase
         {
-            private readonly ImmutableArray<CustomModifier> _refCustomModifiers;
-            private readonly ImmutableArray<CSharpAttributeData> _attributes;
+            private readonly ParameterSymbol _baseParameter;
+            private readonly bool _inheritAttributes;
+
 
             public SynthesizedComplexParameterSymbol(
```

Comment:
>public SynthesizedComplexParameterSymbol( [](start = 12, length = 41)

It looks like an extra empty line is added above this one, CodeFlow shows two empty lines.